classes/plugininfo/cltr.php

```diff
@@ -75,9 +75,9 @@ public function set_enabled(bool $enable) {
     /**
      * Get the collector class for this plugin.
      *
-     * @return collector_base
+     * @return base
      */
-    public function get_collector(): collector_base {
+    public function get_collector(): base {
```

Comment:
Would it be worth having renaming base class to 'collector' and just having base definitions? that way all the things that deal with collectors in aggregate can use `collector` as their param and return types, rather than `base` which feels like a misnomer that could be a little clearer. Perhaps the same for the metric class

Reply:
This naming was accepted in a previous PR, and is not part of this PR. This change is merely fixing a remnant that was not fixed where it should have been. If you think the names should be different then it needs to go in a separate issue.